src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,109 @@
+package com.notably.logic.suggestion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.commons.path.AbsolutePath;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.BlockModel;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+import com.notably.testutil.TypicalBlockModel;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    private static Model model;
+
+    public static Model getModel() {
+        BlockModel blockModel = TypicalBlockModel.getTypicalBlockModel();
+        SuggestionModel suggestionModel = new SuggestionModelImpl();
+        ViewStateModel viewStateModel = new ViewStateModelImpl();
+        model = new ModelManager(blockModel, suggestionModel, viewStateModel);
+
+        return model;
+    }
+
+    public static AbsolutePath getToCs2103t() {
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSuggestionsToCs2103t() {
+        SuggestionItem cs2103t = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation(), null);
+        SuggestionItem cs2103tLect = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut1 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut2 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation(), null);
+
```

Comment:
Also, the more I look at it, the more fragile I feel to not make the `action` parameter of `SuggestionItemImpl`'s constructor do nullity checks. If you need a `Runnable` that does nothing, you can always do `() -> {}`